themes/default/content/blog/pulumi-release-notes-80/index.md

```diff
@@ -0,0 +1,281 @@
+  pulumi:pulumi:Stack: (same)
+    [urn=urn:pulumi:dev::iac-workshop::pulumi:pulumi:Stack::iac-workshop-dev]
+    ~ aws:s3/bucket:Bucket: (update)
+        [id=my-bucket-02d7e7a]
+        [urn=urn:pulumi:dev::iac-workshop::aws:s3/bucket:Bucket::my-bucket]
+        [provider=urn:pulumi:dev::iac-workshop::pulumi:providers:aws::default_1_7_0::229428dd-3bcc-4dcf-ae56-ded0b3b0f322]
+      + website: {
+          + indexDocument: "index.html"
+        }
+    --outputs:--
+  + bucketEndpoint: output<string>
+  ~ bucketName    : "my-bucket-02d7e7a" => output<string>
+    ~ aws:s3/bucketObject:BucketObject: (update)
+        [id=index.html]
+        [urn=urn:pulumi:dev::iac-workshop::aws:s3/bucketObject:BucketObject::index.html]
+        [provider=urn:pulumi:dev::iac-workshop::pulumi:providers:aws::default_1_7_0::229428dd-3bcc-4dcf-ae56-ded0b3b0f322]
+      ~ acl        : "private" => "public-read"
+      ~ contentType: "binary/octet-stream" => "text/html"
+
+Do you want to perform this update? no
+```
+
+After
+
+```console
+Previewing update (dev):
+
+     Type                    Name              Plan       Info
+     pulumi:pulumi:Stack     iac-workshop-dev
+ ~   ├─ aws:s3:Bucket        my-bucket         update     [diff: +website]
+ ~   └─ aws:s3:BucketObject  index.html        update     [diff: ~acl,contentType]
+
+Outputs:
+  + bucketEndpoint: output<string>
+  ~ bucketName    : "my-bucket-02d7e7a" => output<string>
+
+Resources:
+    ~ 2 to update
+    1 unchanged
+
+Do you want to perform this update? details
+  pulumi:pulumi:Stack: (same)
+    [urn=urn:pulumi:dev::iac-workshop::pulumi:pulumi:Stack::iac-workshop-dev]
+    ~ aws:s3/bucket:Bucket: (update)
+        [id=my-bucket-02d7e7a]
+        [urn=urn:pulumi:dev::iac-workshop::aws:s3/bucket:Bucket::my-bucket]
+        [provider=urn:pulumi:dev::iac-workshop::pulumi:providers:aws::default_1_7_0::229428dd-3bcc-4dcf-ae56-ded0b3b0f322]
+      + website: {
+          + indexDocument: "index.html"
+        }
+    ~ aws:s3/bucketObject:BucketObject: (update)
+        [id=index.html]
+        [urn=urn:pulumi:dev::iac-workshop::aws:s3/bucketObject:BucketObject::index.html]
+        [provider=urn:pulumi:dev::iac-workshop::pulumi:providers:aws::default_1_7_0::229428dd-3bcc-4dcf-ae56-ded0b3b0f322]
+      ~ acl        : "private" => "public-read"
+      ~ contentType: "binary/octet-stream" => "text/html"
+
+    --outputs:--
+  + bucketEndpoint: output<string>
+  ~ bucketName    : "my-bucket-02d7e7a" => output<string>
+
+Do you want to perform this update? no
+```
+
+👉 Learn more by reviewing the [Display outputs last in diff view pull request](https://github.com/pulumi/pulumi/pull/10535)
+
+### Support lazy-loading Node modules
+
+We now have opt-in support for generating lazy load code for Node modules. Modules that define functions and resources are loaded lazily when you opt-in. Generated code may use `import type ..` form when importing enums ("useTypeOnlyReferences" opt-in flag in schema).
```

Comment:
Is this being used anywhere yet?  Might be worth noting either that it is being used in Azure Native, or that we plan to use it in Azure Native and other large Pulumi Packages to improve performance when using these libraries?